lib/ibandit/german_details_converter.rb

```diff
@@ -657,138 +657,138 @@ class Rule003101 < BaseRule
-        '538' => '82020086', '540' => '65020186', '541' => '80020087',
-        '545' => '18020086', '546' => '10020890', '547' => '10020890',
-        '548' => '10020890', '549' => '82020087', '555' => '79020076',
-        '560' => '79320075', '567' => '86020086', '572' => '10020890',
-        '580' => '70020270', '581' => '70020270', '601' => '74320073',
-        '602' => '70020270', '603' => '70020270', '604' => '70020270',
-        '605' => '70020270', '606' => '70020270', '607' => '74320073',
-        '608' => '72020070', '609' => '72020070', '610' => '72020070',
-        '611' => '72020070', '612' => '71120077', '613' => '70020270',
-        '614' => '72020070', '615' => '70025175', '616' => '73420071',
-        '617' => '68020186', '618' => '73120075', '619' => '60020290',
-        '620' => '71120077', '621' => '71120077', '622' => '74320073',
-        '623' => '72020070', '624' => '71020072', '625' => '71023173',
-        '626' => '71020072', '627' => '71021270', '628' => '71120077',
-        '629' => '73120075', '630' => '71121176', '631' => '71022182',
-        '632' => '70020270', '633' => '74320073', '634' => '70020270',
-        '635' => '70320090', '636' => '70320090', '637' => '72120078',
-        '638' => '72120078', '640' => '70020270', '641' => '70020270',
-        '643' => '74320073', '644' => '70020270', '645' => '70020270',
-        '646' => '70020270', '647' => '70020270', '648' => '72120078',
-        '649' => '72122181', '650' => '54520194', '652' => '71021270',
-        '653' => '70020270', '654' => '70020270', '655' => '72120078',
-        '656' => '71120078', '657' => '71020072', '658' => '68020186',
-        '659' => '54520194', '660' => '54620093', '661' => '74320073',
-        '662' => '73120075', '663' => '70322192', '664' => '72120078',
-        '665' => '70321194', '666' => '73322380', '667' => '60020290',
-        '668' => '60020290', '669' => '73320073', '670' => '75020073',
-        '671' => '74220075', '672' => '74020074', '673' => '74020074',
-        '674' => '74120071', '675' => '74020074', '676' => '74020074',
-        '677' => '72020070', '678' => '72020070', '679' => '54520194',
-        '680' => '71120077', '681' => '67020190', '682' => '78020070',
-        '683' => '71020072', '684' => '70020270', '685' => '70020270',
-        '686' => '70020270', '687' => '70020270', '688' => '70020270',
-        '689' => '70020270', '690' => '76520071', '692' => '70020270',
-        '693' => '73420071', '694' => '70021180', '695' => '70320090',
-        '696' => '74320073', '697' => '54020090', '698' => '73320073',
-        '710' => '30220190', '711' => '70020270', '712' => '10020890',
-        '714' => '76020070', '715' => '75020073', '717' => '74320073',
-        '718' => '87020086', '719' => '37020090', '720' => '30220190',
-        '723' => '77320072', '733' => '83020087', '798' => '70020270'
+        '168' => '70320090', '169' => '79020076', '170' => '70020270',
```

Comment:
Fixes a warning about a duplicate key when running on Ruby 2.2. (key `'167'` was in here twice before)